### ISSUE TITLE
panel labels are wrong in joined plots

### ISSUE THREAD

**Issue by xiekunwhy** [royfrancis/pophelper#18]
Hi,
Me again. Pophelper is a nice package for visualizing admixture/structure results.
But some small problems still trouble me when I using it in my project.
1) The panel labels are disorderly when K > 10 in plotQ (with imgoutput = "join"), see png in attachment;
here is my code:
library(pophelper)
library(gtools)
sfiles <- list.files(path="AA_noI_admixture_pop-both", full.names=T, pattern = "merge")
slist <- readQ(files=sfiles,indlabfromfile=T)
slist <- slist[match(mixedsort(names(slist)) , names(slist))]
klabs <- paste0("K", sprintf("%02d", 2:(length(slist)+1)))
p <- plotQ(slist[1:20],imgoutput="join",showindlab=T,showlegend=F, width = 25, useindlab = T,
           height=1.6, splab = klabs[1:20], indlabsize=0.5, indlabheight=0.1,indlabspacer = 0.05,
           barbordersize=0,outputfilename="AA_noI_abnormal_2-21_merged",imgtype="png",
           exportplot=T, returnplot=T, sortind = "all", sharedindlab = F, returndata = T)

2) Default color scheme is beautiful when K <= 10, but not so good when K > 10, I found color of Admixture barplot in this paper(http://www.pnas.org/content/110/2/453.full) is nicer than pophelper default, may be you can use it as default. And some pretty color sets for larger K (some times K = 30) are welcomed, but it is difficult for you(and for any others) to prepare.
![aa_noi_abnormal_2-11_merged](https://user-images.githubusercontent.com/17738229/34760433-ca4fd5ac-f61a-11e7-9096-a22cd560ef99.png)
![aa_noi_abnormal_2-21_merged](https://user-images.githubusercontent.com/17738229/34760437-cbf3cdaa-f61a-11e7-9e9f-159485b700c2.png)
![aa_noi_normal_2-10_merged](https://user-images.githubusercontent.com/17738229/34760439-cc37756e-f61a-11e7-9b02-61a0bfd38604.png)
[AA_noI_admixture_pop-both.zip](https://github.com/royfrancis/pophelper/files/1617749/AA_noI_admixture_pop-both.zip)
Best wishes.
Xie Kun

**Comment by royfrancis** [comment]
Hi Xie Kun,
Thank you reporting the labelling issue. It was a bug in the code. I will fix this and release a new version (today or tomorrow).
Regarding the colours, the default colours define up to K=12. The colours from K=12 and higher are not ideal, I agree. But you are free to use your own colours using `clustercol=`. Thanks for suggesting the Morris et. al., paper. They are 10 colours and seem nice when they are separate. But, I am not sure if they work very well when mixed together. See below:

![morris](https://user-images.githubusercontent.com/5083500/34773632-b4282f4e-f60c-11e7-8577-ed1481acdc84.png)

Here are the colours if you are interested:

```
morris_10 <- c("#4d94cc", "#34648a", "#8b658a", "#9acd32", "#cc95cc", "#9acd32", "#8b3a39", "#cd6601", "#cc5c5b", "#8a4500")
```

A nice set of colours you could use is from [Wong,2011](https://www.nature.com/articles/nmeth.1618?WT.ec_id=NMETH-201106). 

![wong](https://user-images.githubusercontent.com/5083500/34773867-7dbb42f6-f60d-11e7-8fce-db82eb783772.png)

Colours below:

```
wong_7 <- c("#000000","#e69f00","#56b4e9","#009e73","#f0e442","#006699","#d55e00","#cc79a7")
```

They are also suitable for colour-blind, but these are only 7 colours.

**Comment by royfrancis** [comment]
Issue with incorrect ordering of panel labels in 'join' plot has been fixed in version `2.2.5`.
